src/get-started/wordcount-example.md

```diff
@@ -79,52 +95,73 @@ The Minimal WordCount pipeline contains five transforms:
 
 1.  A text file `Read` transform is applied to the Pipeline object itself, and produces a `PCollection` as output. Each element in the output PCollection represents one line of text from the input file. This example happens to use input data stored in a publicly accessible Google Cloud Storage bucket ("gs://").
 
-	```java
-	        p.apply(TextIO.Read.from("gs://apache-beam-samples/shakespeare/*"))
-	```
+    ```java
+    p.apply(TextIO.Read.from("gs://apache-beam-samples/shakespeare/*"))
+    ```
+
+    ```py
+    {% github_sample /apache/beam/blob/python-sdk/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_read
+    %}```
 
 2.  A [ParDo]({{ site.baseurl }}/documentation/programming-guide/#transforms-pardo) transform that invokes a `DoFn` (defined in-line as an anonymous class) on each element that tokenizes the text lines into individual words. The input for this transform is the `PCollection` of text lines generated by the previous `TextIO.Read` transform. The `ParDo` transform outputs a new `PCollection`, where each element represents an individual word in the text.
 
-	```java
-	        .apply("ExtractWords", ParDo.of(new DoFn<String, String>() {
-	            @ProcessElement
-	            public void processElement(ProcessContext c) {
-	                for (String word : c.element().split("[^a-zA-Z']+")) {
-	                    if (!word.isEmpty()) {
-	                        c.output(word);
-	                    }
-	                }
-	            }
-	        }))
-	```
+    ```java
+    .apply("ExtractWords", ParDo.of(new DoFn<String, String>() {
+        @ProcessElement
+        public void processElement(ProcessContext c) {
+            for (String word : c.element().split("[^a-zA-Z']+")) {
+                if (!word.isEmpty()) {
+                    c.output(word);
+                }
+            }
+        }
+    }))
+    ```
+
+    ```py
+    # The Flatmap transform is a simplified version of ParDo.
```

Comment:
I would have preferred it to be text but we don't have the ability to put text inside the language toggles yet so I put it in as a comment temporarily.